### ISSUE TITLE
[Android][OpenGL&Vulkan] System Font? Too much large and uniform. 

### ISSUE THREAD

**Issue by nassau-tk** [hrydgard/ppsspp#12764]
### What happens?
Some Information screen
Some Warning screen
Some Error screen
Some Save data screen
Theese Font (Text) are too much large more than real PSP.
It isn't uniform on savedata screen.
I don't know the issue will happen only Japanese language.

### What *should* happen?
It'll be the like a proper size.

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.

PPSSPP Ver. : 1.9.3-561
OS: Android 9.0
DeviceName: SONY SO-02K / Xperia XZ1 Compact (Japan)
CPU: Snapdragon835 (Octa-core)
GPU: Adreno 540
Resolution: 720×1280
RAM: 4GB
ROM:32GB
ExternalStrage: microSDXC 512GB

［Real PSP Tekken6 AutoSave info］
![DSC_0081](https://user-images.githubusercontent.com/48179091/77598543-71268d80-6f45-11ea-89a7-71cedc05c650.JPG)

［PPSSPP Tekken6 AutoSave info］
![Screenshot_20200326-091844](https://user-images.githubusercontent.com/48179091/77598587-961b0080-6f45-11ea-8c61-24fb9e1d1409.png)

［Real PSP Dunball Senki Savedata］
![DSC_0082](https://user-images.githubusercontent.com/48179091/77598675-c6629f00-6f45-11ea-82f8-4dd0bf395cdb.JPG)

［PPSSPP Dunball Senki Savedata No.1］
![Screenshot_20200326-092914](https://user-images.githubusercontent.com/48179091/77598482-49372a00-6f45-11ea-855f-84716b724cbe.png)

［PPSSPP Dunball Senki Savedata No.2］
![Screenshot_20200326-092926](https://user-images.githubusercontent.com/48179091/77598797-1c374700-6f46-11ea-8bf9-0cc07565c497.png)

**Comment by Panderner** [comment]
Is this spacing issue?

**Comment by nassau-tk** [comment]
Space & Size both.
May, be l think.

**Comment by hrydgard** [comment]
We use system font rendering now (to avoid the issues of missing characters and so on) so the match will not be exact, but it does indeed look like we need to go down a point size or two.

**Comment by nassau-tk** [comment]
I got a screenshot from real PSP.
by reference.

![HUD-FINAL_Tekken6](https://user-images.githubusercontent.com/48179091/77627089-b371be00-6f89-11ea-95cf-22e12276cfc7.png)


![HUD-FINAL_dunballsenki](https://user-images.githubusercontent.com/48179091/77626221-64775900-6f88-11ea-978a-65f734a194df.png)

**Comment by hrydgard** [comment]
Reduced the size by a couple of points, how is this now?

**Comment by Panderner** [comment]
X enter text is overlapped by a line in echochrome
![Screenshot_2020-03-29-00-27-49-52_2f85358b2198d26f8aca533d68bee793](https://user-images.githubusercontent.com/37503397/77828043-49f1db00-7154-11ea-8578-00349cb66fcd.png)

**Comment by hrydgard** [comment]
Hm, yeah, looks like the line positioning could be tweaked, and should maybe reduce size by another point.

**Comment by Panderner** [comment]
After v1.9.3-584-g620aa50cf the spaces still not fixed.
![Screenshot_2020-03-29-20-34-35-10_2f85358b2198d26f8aca533d68bee793](https://user-images.githubusercontent.com/37503397/77849248-e079d700-71fc-11ea-9d26-1141f47b9bff.png)

**Comment by hrydgard** [comment]
Yeah, we need to shrink the font a little more, and also I'm considering squashing it a little bit vertically, because the PSP's font is not very tall. Although that might not look very good for Chinese, Japanese and Korean characters.

**Comment by unknownbrackets** [comment]
I think we should allow the message to display wider, because the proportions of the font we're using are different.  As has been widely covered, the PSP's system font has somewhat unusual proportions.

-[Unknown]

**Comment by hrydgard** [comment]
Yeah, that would also help.

**Comment by Panderner** [comment]
And here's a Bleach Heat The Soul 7 after data install:
![Screenshot_2020-04-24-21-31-20-95_2f85358b2198d26f8aca533d68bee793](https://user-images.githubusercontent.com/37503397/80218041-39f5ea00-8673-11ea-998b-425d675c8a73.png)

**Comment by hrydgard** [comment]
Actually for that case there seems to be a bug where we show an OK button, even though there isn't one on a real PSP. That causes the layout condition (in addition to the text still being a little too large).

**Comment by Panderner** [comment]
![IMG_20200818_022509](https://user-images.githubusercontent.com/37503397/90430614-5aeabb00-e0fa-11ea-9bc1-54bca30ef9a4.jpg)
After v1.10.3-129-g44954cff3 Still not correct for empty spaces

**Comment by nassau-tk** [comment]
This is an amature's question.

If revert the using system font files to PGF files on warning screen then what will happen?
Because,we modified PGF files.
(I'm slightly interested.)

**Comment by hrydgard** [comment]
We don't have a way to do that currently. These dialogs are not created from PSP code, so can't directly use the PGF library. It would be quite a lot of work to set it up...

**Comment by nassau-tk** [comment]
Oh...l understood about it.
Thank you.

**Comment by hrydgard** [comment]
I'm gonna bump further work on this to 1.12, things are at least improved.

**Comment by nassau-tk** [comment]
Yes,We are not feeling tnuseful on it.
So,Priority is low l think.
